example_ws/src/example_pkg/src/example_pkg/utils.py

```diff
@@ -12,3 +12,17 @@ def foo_bar():
         return True
     else:
         return False
+
+
+def foo_bar_2():
+    if 4 % 2 == 0:
+        return True
+    else:
+        return False
+
+
+def foo_bar_3():
+    if 4 % 2 == 0:
+        return True
+    else:
+        return False
```

Comment:
I added two util functions that dont get called from the rostest, just the nosetests. 

These methods are not covered as part of coverage even though one of them should be since it gets called in the test_utils.py file. :(
https://codecov.io/gh/SarvagyaVaish/ros_ci_and_coverage/src/f0720a057d0c60d06027bdc6b468a85ce9612419/example_ws/src/example_pkg/src/example_pkg/utils.py